pioneer/packages/app-tokenomics/src/lib/getContentCuratorData.ts

```diff
@@ -0,0 +1,58 @@
+import { ApiPromise } from '@polkadot/api';
+import { Reward } from './getTokenomicsData';
+
+type ContentCurator = {
+  reward_relationship: number;
+  role_stake_profile: {
+    stake_id: number;
+  };
+}
+
+const getCurators = async (api: ApiPromise): Promise<Array<ContentCurator>> => {
```

Comment:
This one is incredibly nice, but it would take a very long time to implement all the logic. I'll keep this one(and some of the other tips you've had that I may not have commented on) in mind when adapting this for `Alexandria`.